### PR TITLE
Renaming the "Body" objects to MixedRealityPlayspace in example maps

### PR DESCRIPTION
**Pull request by Ecnassianer**
This is a follow-up to my code PR that initializes a MixedRealityPlayspace instead of using a "Body":
https://github.com/Microsoft/MixedRealityToolkit-Unity/issues/2719

This should probably wait for the code PR to go through before it's merged, but I wanted to do it separately in case there were any merge conflicts, we can throw this PR away and redo it easily.

### CODE REVIEW

**Comment by davidkline-ms** [comment]
@StephenHodgson, why is this marked blocked?

**Comment by keveleigh** [comment]
@davidkline-ms this contains the changes also present in #2773, so that should be merged first.

**Comment by davidkline-ms** [comment]
Perfect! Thanks.

**Comment by davidkline-ms** [comment]
@Ecnassianer, can you also update instances of the MainCamera being labeled as Head?

**Comment by StephenHodgson** [comment]
Yeah I think you just need to merge the latest changes from dev into your branch Eric.

File: Assets/MixedRealityToolkit/_Core/Devices/MixedReality/WindowsMixedRealityController.cs [hunk 1]
```diff
@@ -5,6 +5,7 @@
 using Microsoft.MixedReality.Toolkit.Core.Definitions.InputSystem;
 using Microsoft.MixedReality.Toolkit.Core.Definitions.Utilities;
 using Microsoft.MixedReality.Toolkit.Core.Interfaces.InputSystem;
+using Microsoft.MixedReality.Toolkit.Core.Managers;
```
Comment (StephenHodgson) [inline Assets/MixedRealityToolkit/_Core/Devices/MixedReality/WindowsMixedRealityController.cs]: What changed here? Do we actually need to reference this?

File: Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs [hunk 2]
```diff
@@ -146,6 +146,7 @@ private void Initialize()
                 }
             }
 #endif
+            EnsureMixedRealityRequirements();
```
Comment (StephenHodgson) [inline Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs]: Could we rename this to be something a bit more specific?

**Comment by Ecnassianer** [comment]
This PR ended up with conflicts. I'm going to rebase/cherrypick these content changes into a fresh branch that doesn't have a merge history. Closing this PR.

**Pull request closed without merging**